client/src/app/site/common/components/start/start.component.ts

```diff
@@ -76,7 +75,23 @@ export class StartComponent extends BaseModelContextComponent implements OnInit
      */
     public ngOnInit(): void {
         super.setTitle(`Home`);
+        this.refresh();
+    }
+
+    private refresh(): void {
         this.requestUpdates();
+        this.updateSubscription(
+            WELCOME_TITLE_SUBSCRIPTION_NAME,
+            this.meetingSettingsService.get(`welcome_title`).subscribe(welcomeTitle => {
+                this.startContent.welcome_title = welcomeTitle;
+            })
+        );
+        this.updateSubscription(
+            WELCOME_TEXT_SUBSCRIPTION_NAME,
+            this.meetingSettingsService.get(`welcome_text`).subscribe(welcomeText => {
+                this.startContent.welcome_text = this.translate.instant(welcomeText);
+            })
+        );
```

Comment:
That might not be necessary. You can simply get the observable from the service and give it to the UI directly using async pipe and translate pipe. Youd also gain a translateable welcome text on the fly